lib/pool.js

```diff
@@ -33,15 +25,15 @@ function Pool(url, options = {}) {
   )
 }
 
-Pool.prototype.request = function (method, path, data) {
+Pool.prototype.request = function (method, path, headers, data) {
```

Comment:
I'm curious if there's any reason `headers` has to be the third argument here, given that we switch them around in `get` `post` etc below?